src/ContextMenuTrigger.js

```diff
@@ -28,6 +28,7 @@ export default class ContextMenuTrigger extends Component {
         renderTag: 'div'
     };
 
+    static touchHandled = false;
```

Comment:
This shouldn't be static but a regular instance property if you address it with `this` below. i.e. just remove the `static` keyword and all is fine.